keyboards/darkproject/kd87a_bfg_edition/info.json

```diff
@@ -0,0 +1,278 @@
+            { "flags": 4, "matrix": [6, 1], "x": 125, "y": 25 },
+            { "flags": 4, "matrix": [10, 2], "x": 135, "y": 25 },
+            { "flags": 4, "matrix": [6, 5], "x": 155, "y": 25 },
+            { "flags": 4, "matrix": [7, 5], "x": 165, "y": 25 },
+            { "flags": 4, "matrix": [2, 5], "x": 175, "y": 25 },
+
+            { "flags": 8, "matrix": [2, 1], "x": 0, "y": 35 },
+            { "flags": 4, "matrix": [1, 2], "x": 17.5, "y": 35 },
+            { "flags": 4, "matrix": [2, 2], "x": 27.5, "y": 35 },
+            { "flags": 4, "matrix": [3, 2], "x": 37.5, "y": 35 },
+            { "flags": 4, "matrix": [4, 2], "x": 47.5, "y": 35 },
+            { "flags": 4, "matrix": [4, 3], "x": 57.5, "y": 35 },
+            { "flags": 4, "matrix": [5, 3], "x": 67.5, "y": 35 },
+            { "flags": 4, "matrix": [5, 2], "x": 77.5, "y": 35 },
+            { "flags": 4, "matrix": [6, 2], "x": 87.5, "y": 35 },
+            { "flags": 4, "matrix": [7, 2], "x": 97.5, "y": 35 },
+            { "flags": 4, "matrix": [8, 2], "x": 107.5, "y": 35 },
+            { "flags": 4, "matrix": [8, 3], "x": 117.5, "y": 35 },
+            { "flags": 4, "matrix": [10, 4], "x": 127.5, "y": 35 },
+
+            { "flags": 4, "matrix": [0, 0], "x": 0, "y": 45 },
+            { "flags": 4, "matrix": [1, 4], "x": 22.5, "y": 45 },
+            { "flags": 4, "matrix": [2, 4], "x": 32.5, "y": 45 },
+            { "flags": 4, "matrix": [3, 4], "x": 42.5, "y": 45 },
+            { "flags": 4, "matrix": [4, 4], "x": 52.5, "y": 45 },
+            { "flags": 4, "matrix": [4, 5], "x": 62.5, "y": 45 },
+            { "flags": 4, "matrix": [5, 5], "x": 72.5, "y": 45 },
+            { "flags": 4, "matrix": [5, 4], "x": 82.5, "y": 45 },
+            { "flags": 4, "matrix": [6, 4], "x": 92.5, "y": 45 },
+            { "flags": 4, "matrix": [7, 4], "x": 102.5, "y": 45 },
+            { "flags": 4, "matrix": [8, 5], "x": 112.5, "y": 45 },
+            { "flags": 4, "matrix": [9, 1], "x": 122.5, "y": 45 },
+            { "flags": 4, "matrix": [3, 5], "x": 165, "y": 45 },
+
+            { "flags": 4, "matrix": [0, 6], "x": 0, "y": 55 },
+            { "flags": 4, "matrix": [9, 0], "x": 12.5, "y": 55 },
+            { "flags": 4, "matrix": [9, 3], "x": 25, "y": 55 },
+            { "flags": 4, "matrix": [9, 4], "x": 37.5, "y": 55 },
+            { "flags": 4, "matrix": [9, 5], "x": 100, "y": 55 },
+            { "flags": 4, "matrix": [9, 2], "x": 112.5, "y": 55 },
+            { "flags": 4, "matrix": [8, 4], "x": 125, "y": 55 },
+            { "flags": 4, "matrix": [0, 4], "x": 137.5, "y": 55 },
+            { "flags": 4, "matrix": [0, 3], "x": 155, "y": 55 },
+            { "flags": 4, "matrix": [7, 3], "x": 165, "y": 55 },
+            { "flags": 4, "matrix": [0, 5], "x": 175, "y": 55 }
+        ]
+    },
+    "url": "",
+    "usb": {
+        "device_version": "0.0.3",
+        "pid": "0xE393",
+        "vid": "0x342D"
```

Comment:
This is the Westberry VID.  Do you have permission to use it?

Reply:
@drashna yes we confirmed with WB to use this VID.